### PR TITLE
Add page contents

### PR DESCRIPTION
**Pull request by iBel251**
- Add images and texts for the website contents.
- Fix the 'more' and 'less' buttons on the index page.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,52 @@
-## Welcome to GitHub Pages
+# The Grand Auction (Capstone 1)
 
-You can use the [editor on GitHub](https://github.com/iBel251/grand-auction/edit/main/README.md) to maintain and preview the content for your website in Markdown files.
+## Description
```
Comment (Deepakdanger) [inline README.md:8]: - [ ] [ OPTIONAL] It is nice to have some project pics in the readme file.

File: README.md [hunk 2]
```diff
@@ -1,37 +1,52 @@
...
+## Built With
 
-```markdown
-Syntax highlighted code block
+- Major languages:- Html, Css and Javascript
 
-# Header 1
-## Header 2
-### Header 3
+- Frameworks:- VS code
```
Comment (Deepakdanger) [inline README.md:26]: - [ ] **Required** It is nice to have a `Getting started` section with necessary details. Refer [here](https://github.com/microverseinc/readme-template/blob/master/README.md)
- Prerequisites
- Setup
- Install

File: index.html [hunk 3]
```diff
@@ -80,7 +80,8 @@ <h2>Featured Items</h2>
         <div class="item-cards-container">
 
         </div>
-        <div class="morebtn" onclick="moreCards()">MORE <i class="fa-solid fa-angle-down"></i></div>
+        <div class="morebtn more" onclick="moreCards()">MORE <i class="fa-solid fa-angle-down"></i></div>
+        <div class="morebtn less hide-item" onclick="moreCards()">LESS <i class="fa-solid fa-angle-up"></i></div>
     </section>
     <section id="partners" class="footer-partners">
```
Comment (Deepakdanger) [inline index.html:86]: - [ ] **Required** In the `Partner` section there should be the logo image of the partner, Kindly remove the `daumkakao` text & add the logo there.
- Your design
Screenshot from 2022-04-14 17-32-45

- Required design 
Screenshot from 2022-04-14 17-33-02

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @iBel251, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :heavy_check_mark: 
- Your background is different from the requirement, but it's similar to your website content. :+1: 

## Suggested changes
- [ ] On the `About` page the content of the top section should have been underlined with an arrow icon. Kindly add this.
-  Your design 
Screenshot from 2022-04-14 17-33-52
- Required design
Screenshot from 2022-04-14 17-45-46
- [ ] Kindly update your live link.
- [ ] The student recorded a video presenting the features of the project.
- The maximum length of the video is 5 minutes.
- The video contains a clear and objective description about the project in professional English. The topic and purpose of the project are made clear.
- The video contains a demo of the project, showing the pages in different screen sizes.
- The video contains a highlight about a piece of the code the student finds interesting or is proud of making.
- [ ] _Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @iBel251 😃,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlight**
- Well done working on the required changes
- Great design following

Cheers and Happy coding!👏👏👏

**OPTIONAL**
- Please see the recommended optional changes from the previous review
- Kindly fix the MIT link for a completely professional README file

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**